examples/rllab/green_wave.py

```diff
@@ -33,7 +33,7 @@ def gen_edges(row_num, col_num):
 def get_flow_params(v_enter, vehs_per_hour, col_num, row_num,
                     additional_net_params):
     initial_config = InitialConfig(
-        spacing="uniform", lanes_distribution=float("inf"), shuffle=True)
+        spacing="custom", lanes_distribution=float("inf"), shuffle=True)
```

Comment:
what's the reasoning for this?

Reply:
The grid network actually overrides the uniform starting position method, which causes issues once the kernel scenario gets introduced, so I just replaced `gen_uniform_start_pos` with `gen_custom_start_pos` and modified everything accordingly. Ignoring what might come with the scenario kernel, I still think this is a good idea.